test/verify/check-sosreport

```diff
@@ -71,6 +71,7 @@ only-plugins=release,date,host,cgroups,networking
         b.set_input_text("#sos-dialog .pf-c-form__group:contains(Report label) input", "mylabel")
         b.set_input_text("#sos-dialog .pf-c-form__group:contains(Encryption passphrase) input", "foobar")
         b.set_checked("#sos-dialog .pf-c-check:contains(Obfuscate) input", True)
+        b.set_checked("#sos-dialog .pf-c-check:contains(Use verbose logging) input", True)
```

Comment:
Let's please test the default as well, not *only* verbose mode.

Reply:
Yes, good point.

Reply:
Done.